jsamchineme/send-it
# Add display of user profile info

#### What does this PR do?
- Implement display of user profile info
#### Description of Task to be completed?
- fetch user data through the service API
- Display profile data on the page header
- Display profile data and summary of order history
#### How should this be manually tested?
- pull branch and run _npm run start:dev_
- navigate to http://localhost:3001/login and login using _email: 'jsamchineme@gmail.com', password: 'secretpass'_
- see the user data on the home page
#### Any background context you want to provide?
- None
#### What are the relevant pivotal tracker stories?
- https://www.pivotaltracker.com/story/show/162829515
#### Screenshots (if appropriate)
None
#### Questions:`
None
